plugins/base/src/main/kotlin/transformers/pages/sourcelinks/SourceLinksTransformer.kt

```diff
@@ -92,8 +93,9 @@ class SourceLinksTransformer(val context: DokkaContext) : PageTransformer {
     }
 
     private fun PsiElement.lineNumber(): Int? {
+        val skippedKDoc = children.firstOrNull { it !is KDoc } ?: this
```

Comment:
Will this work for Java sources the same way that it will for Kotlin? Shouldn't there also be something like `it !is JavaDoc` (or a common parent)?